### ISSUE TITLE
Allow to translate country names

### ISSUE THREAD

**Issue by TheZoker** [bytefury/crater#418]
**Is your feature request related to a problem? Please describe.**
Currently the country names in invoices use english as a language. It would be very nice, if the country would use the translated name. So `Germany` becomes `Deutschland`. 

**Describe the solution you'd like**
Use translated country names

**Describe alternatives you've considered**
none

**Comment by LivingWithHippos** [comment]
As a temporary solution I used omnidb to interface with the database and manually update the country name

```
UPDATE countries
SET name = 'Italia'
WHERE name = 'Italy';
```

**Comment by albertvisuals** [comment]
Yes, we need the possibility to translate country names. Furthermore Date/time is not translated (localised) as well.
